tests/pytests/unit/utils/test_minions.py

```diff
@@ -65,7 +65,7 @@ def test_validate_tgt_should_return_false_when_no_valid_minions_have_been_found(
         "salt.utils.minions.CkMinions.check_minions", autospec=True, return_value={}
     ):
         result = ckminions.validate_tgt("fnord", "fnord", "fnord", minions=[])
-        assert result is False
```

Comment:
FWIW this test should either be renamed or completely removed - this was part of the CVE check to ensure that we weren't validating minions that weren't valid.

Reply:
Agreed, I've renamed the test and also added a doc string to help explain what's going on.